docs/Language.md

```diff
@@ -0,0 +1,64 @@
+The following guidelines for writing copy are intended to create a consistent style and voice for the intelliHR product.
+
+## Language
+* **Keep text short and simple** to make text accessible to people of all backgrounds
+    * **Do**: This person doesn't have a job record
+    * **Don't**: This individual whose digital profile you are viewing does not appear to have a job record
+
+* **Use industry standard proper nouns** as feature names
+    * **Do**:  Private Notes
+    * **Don't**: Diary Notes
+
+* **Use a friendly tone and avoid niceties** to appear more conversational
+    * **Do**:  Nice! You've completed all of your outstanding tasks.
+    * **Don't**: Task completed successfully. All outstanding tasks have been completed.
+
+* **When to use 'My' and 'Your'**
+    * **You/Your**: When asking questions, giving instructions, or describing things to the user. Imagine writing from the perspective of a personal assistant.
+    * **Me/My**: When the user is interacting with the product on things like buttons- ONLY if absolutely necessary for clarity. _First person is not preferred._
+
+* **Dates**
+    * **Do**: Always respect the user's shorthand date format (DD/MM/YY, MM/DD/YY, YY/MM/DD, etc.)
+    * **Do**: Use `d MMM YYYY` (eg: 7 Jul 1977) in contexts where a more conversational date is required
+
+* **Use input agnostic language** where possible instead of input specific terminology
+    * **Do**: Pick your preferred email address
+    * **Don't**: Click on your preferred email address
+
+* **Action words should follow the following rules** 
+    * **Create/Delete**: When a user will be creating/deleting a system object like a Self-Service Action, Pulse, or Person
+    * **Add/Remove**: When a user is adding/removing additional data to a system object like a Phone Number to a Person, or Measurements to a Goal
+    * **Update**: When a user is editing a system object
+    * **Exceptions**: Occasionally there will be more context appropriate action words such as `Set Goal` and `Issue Task` which are preferred to the above. The appropriate destructive action word should still be used.
+    * **Page Titles and Submit Buttons**: For titles and buttons, use the following format `[Action Name] [System Object Name]`. For example: `Create Person`, `Add Phone Number`, `Set Goal`. On edit forms the submit button should always be `Save`
+    * **Don't**: Use the word `new` on creation actions. It is implied.
+
+## Capitalization and Punctuation
+* **Use Title Case for buttons, page titles, menu items, and labels** - most words are capitalized
+    * **Example**: Set a Goal
+    * **Do Capitalize**: The first letter of nouns, adjectives, verbs, adverbs, and pronouns
+    * **Do Capitalize**: The first and last words of the text element
```

Comment:
Are you saying we'd have text like:

'Hello this is my Button'

Seems kinda odd

Reply:
Well you're never going to have a button actually labelled that but it would be something like `Hello This is My Button` or `Hello This Is My Button`. The general rule with title text is that everything is caps except for articles and short prepositions - maybe I can reword this to make it make more sense. This rule is in cases where you would start or end the the label with an article/preposition like `A Goal` instead of `a Goal`. 

Also since your example is also more conversational than just being a label (which would just be `My Button`) - you'd be better off with sentence case anyway: `Hello this is my button`